pkg/gui/controllers/files_controller.go

```diff
@@ -1009,10 +1009,14 @@ func normalisedSelectedNodes(selectedNodes []*filetree.FileNode) []*filetree.Fil
 
 func isDescendentOfSelectedNodes(node *filetree.FileNode, selectedNodes []*filetree.FileNode) bool {
 	for _, selectedNode := range selectedNodes {
+		if selectedNode.IsFile() {
+			continue
+		}
+
 		selectedNodePath := selectedNode.GetPath()
 		nodePath := node.GetPath()
 
-		if strings.HasPrefix(nodePath, selectedNodePath) && nodePath != selectedNodePath {
+		if strings.HasPrefix(nodePath, selectedNodePath+"/") {
```

Comment:
Tested on Windows, path names will use / instead of \.

Reply:
To recap the logic here for posterity:
* the first part which checks if the selected node is a file is there to ensure that if you have a file named 'a' and a file named 'aa', that the latter is not considered a descendant of the former. In short: if you're a file, it's not possible to have descendants so we can early exit
* The second part is for when you have one directory named 'a' and another named 'aa', using the same logic as above.

Is that right?

Reply:
Yes!